### DISCUSSION TITLE
I can't get domain from name.com

### DISCUSSION THREAD

**Discussion by Zeus0x01** [community/community#140014] [GitHub Education]
### Hi everyone,

when i get the domain from name.com it does not make the payment and the support told me you're not full time student

**Comment by ferid333** [comment]
Hi @Zeus0x01 

 If you're applying for a student discount or offer, make sure to double-check the documentation you submitted. Sometimes they need specific proof, like a current student ID or an official enrollment letter.

If you've already submitted that and are still having issues, I’d suggest reaching out to support again with updated documents or clarification. They might just need a little more information to confirm your student status.

**Comment by davevad93** [comment]
Hi @Zeus0x01 ,

If you are a verified student and your benefits are available (which seems to be the case since you got the PRO badge) you should have access to `name.com` because it's one of the available offers in the `Student Developer Pack`. Contact support by raising a ticket [here](https://support.github.com/contact/education), select 
"`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
